atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -73,13 +73,15 @@ def __init__(self, tasker, workflow, koji_hub=None, koji_ssl_certs_dir=None,
         self._poll_start = None
 
     def run(self):
-        if not self.workflow.builder.base_from_scratch:
-            nvr = self._base_image_nvr = self.detect_parent_image_nvr(
-                self.workflow.builder.base_image,
-                inspect_data=self.workflow.builder.base_image_inspect,
-            )
-            if nvr:
-                self._base_image_build = self.wait_for_parent_image_build(nvr)
+        if self.workflow.builder.base_from_scratch or self.workflow.builder.custom_base_image:
+            return
```

Comment:
Should we log INFO here that this plugin was skipped?

Reply:
we don't want to skip this plugin for base_from_scratch, the only thing we are skipping in that case is checking base_image since that is 'FROM scratch' but we still want to perform check for other parents

Reply:
Fixed.